Week7/app/src/main/java/com/kulloveth/covid19virustracker/worker/WorkerUtil.kt

```diff
@@ -0,0 +1,95 @@
+package com.kulloveth.covid19virustracker.worker
```

Comment:
Really nice work on the notification and I see this is where you check for the network connectivity.